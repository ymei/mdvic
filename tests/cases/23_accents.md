Inline: $\hat{x}$, $\bar{y}$, $\tilde{z}$, $\vec{v}$, and text $\hat{ab}$.

