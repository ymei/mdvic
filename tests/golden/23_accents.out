Inline: x̂, ȳ, z̃, v⃗, and text ab̂.
